README.md

```diff
@@ -54,7 +54,7 @@ import { generateImage } from "jsdom-screenshot";
 
 // add some content to jsdom (this could also be React or any other library!)
 const div = document.createElement("div");
-div.innerText = "Hello World";
+div.innerHTML = "Hello World";
```

Comment:
`innerText` should stay here https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText

Reply:
I don't know how you tested but it fails on my local due to innerText is not implemeted in jsdom, see discussion here: https://github.com/jsdom/jsdom/issues/1245

Reply:
Ah I don't remember how I tested it. With that context, the change makes sense. Thanks!